backend/src/main/java/fr/anettehaferkorn/backend/utils/DBService.java

```diff
@@ -0,0 +1,30 @@
+package fr.anettehaferkorn.backend.utils;
+
+import fr.anettehaferkorn.backend.Recommendations.model.WineGrape;
+import fr.anettehaferkorn.backend.Recommendations.repo.WineGrapeRepository;
+import fr.anettehaferkorn.backend.wineFinderAPI.model.WineBottleDTO;
+import fr.anettehaferkorn.backend.wineFinderAPI.repo.WineBottlesRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class DBService {
+
+    WineGrapeRepository wineGrapeRepository;
+    WineBottlesRepo findYourWineRepo;
+
+    @Autowired
```

Comment:
The `@Autowired` is not necessary. If you annotate the class with `@RequiredArgsConstructor` and make the fields final (and I would also make them private), you could delete the constructor.